tool/make-native-distribution.sh

```diff
@@ -67,25 +84,19 @@ mkdir -p "$release_home"
 export TRUFFLERUBY_RESILIENT_GEM_HOME=true
 
 # Build
-
-# NOTE: The 3 lines below are no longer needed in the next release
-# Build Sulong first so that TRUFFLERUBY-ZIP includes sulong-libs
-cd ../sulong
-mx build
-cp -r mxbuild/sulong-libs ../truffleruby/lib/cext
-
 cd ../truffleruby
+mx sversions
 mx build
 
 cd ../graal/substratevm
 mx build
 
 # Build image
-mx fetch-languages --llvm --ruby
+mx fetch-languages --Language:llvm --Language:ruby
 
-# Smaller default heap so it fits in Travis CI containers
-./native-image -no-server --llvm --ruby \
+./native-image --no-server --Language:llvm --Language:ruby \
   -H:Path="$build_home/bin" -H:Name=native-ruby \
+  -H:Class=org.truffleruby.launcher.RubyLauncher \
```

Comment:
I think we should de-duplicate this and use jt native build here, with #1254

Reply:
Yes, it's also a nice opportunity to test `jt build native` in the gate.
I'll add that, but I'll need an extra `--no-jvmci` flag in `jt build native`.
Now that we can build from any TruffleRuby commit, it doesn't seem so useful to avoid relying on jt.
But it would mean it will be impossible to build a native dist if `jt build native` is broken (shouldn't be, since it will be gated, but the truffle/svm import could change if we use graalvm tags).